integration-tests/hibernate-reactive-orm-compatibility/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=hibernate_orm_test
+quarkus.datasource.password=hibernate_orm_test
+
+# Hibernate config
+#quarkus.hibernate-orm.log.sql=true
+quarkus.hibernate-orm.database.generation=drop-and-create
+
+# Reactive config
+quarkus.datasource.reactive=true
```

Comment:
Is this required? Ideally it would not be, if it is we should probably file a bug/improvement.

Reply:
> you're referring to `quarkus.datasource.reactive` right?

Yes. Don't know why GitHub is highlighting lines above that, maybe I made a mistake when selecting the lines.

Reply:
Removed the latter configuration key